ops/jenkins/bfd-build-platinum/Jenkinsfile

```diff
@@ -9,6 +9,70 @@
 // These variables are accessible throughout this file (except inside methods and classes).
 def goldAmiId
 def platinumAmiId
+def currentStage
+
+def buildPlatinum(String platinumId = '', String goldId = '') {
+    // Retrieve latest platium and assign the sourceAmiId tag value with the latest gold AMI ID.
+    withCredentials([file(credentialsId: 'bfd-vault-password', variable: 'vaultPasswordFile')]) {
+        // packer is always run from $repoRoot/ops/ansible/playbooks-ccs
+        dir('ops/ansible/playbooks-ccs') {
+            env.goldAmi = goldId
+            echo "Using the folllowing gold image: ${env.goldAmi}"
+
+            sh '''
+            packer build -color=false -var vault_password_file="$vaultPasswordFile" \
+            -var source_ami="${goldAmi}" \
+            -var subnet_id=subnet-092c2a68bd18b34d1 \
+            ../../packer/build_bfd-platinum.json
+            '''
+        }
+    }
+}
+
+// send notifications to slack
+def sendNotifications(String buildStatus = '', String stageName = '') {
+    // build colors
+    def colorMap = [:]
+    colorMap['SUCCESS']  = '#00FF00'
+    colorMap['SUCCESS LATEST IMAGE']  = '#00FF00'
+    colorMap['SUCCESS NEW PLATINUM']  = '#00FF00'
+    colorMap['ABORTED']  = '#6A0DAD'
+    colorMap['UNSTABLE'] = '#FFFF00'
+    colorMap['FAILED']   = '#FF0000'
+    def buildColor = colorMap[buildStatus]
+    buildColor = buildColor ?: '#FF0000' // default to red
+
+    // prettyfi messages
+    def msg = ''
+    switch (buildStatus) {
+        case 'UNSTABLE':
+        case 'SUCCESS LATEST IMAGE':
+            msg = 'COMPLETED SUCCESSFULLY WITH THE LATEST PLATINUM IMAGE'
```

Comment:
I think this needs to explicitly communicate that the job was successful and that it didn't need to do anything, right?

Reply:
Yes, fixing